scripts/deploy-cluster.sh

```diff
@@ -113,6 +113,7 @@ kubectl wait deployment -n kube-system coredns --for condition=Available=True --
 echo ""
 echo "deploy Openshift console (with forklift proxy, tls: false, auth: flase, port: 30088)..."
 kubectl apply -f ${script_dir}/yaml/openshift-console.yaml
+kubectl create namespace konveyor-forklift
```

Comment:
do we need to create it here? `deploy-forklift` script seems a better place i.e  we have there `FORKLIFT_NAMESPACE` var

Reply:
makes sense, I'll take it out

Reply:
removed :+1: